data/news/platform/ocaml-4.14.1.rc1.md

```diff
@@ -0,0 +1,149 @@
+---
+title: OCaml 4.14.1 - First release candidate
```

Comment:
```suggestion
title: OCaml 4.14.1 - First Release Candidate
```
Also, perhaps just “Release Candidate,” as David said there probably won’t be another one.

Reply:
Good point.

Reply:
It's best if this is capitalised, as that's correct in title case